app/build.gradle

```diff
@@ -24,4 +24,5 @@ dependencies {
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.2.1'
     compile 'com.android.support:support-v4:24.0.0-alpha1'
+    compile 'net.danlew:android.joda:2.9.4.1'
```

Comment:
Eh I'll just leave the other one - you have to initialize the android library somewhere and that might make workshop students think you have to initialize all libraries you add.